pages/13.builders/docs.en.md

```diff
@@ -68,7 +68,82 @@ From the 3.3 General Availability release there will be three email templates th
 
 ### Themes
 
-If you search through the list of available themes, the new MJML themes `Brienz`, `Paprika` and `Confirm Me` will not be available until you enable the GrapesJS plugin. 
+If you search through the list of available themes, the new MJML themes `Brienz`, `Paprika` and `Confirm Me` will not be available until you enable the GrapesJS plugin.
+
+### Creating new Themes
+
+#### Why to use email templates?
+The GrapesJs makes possible to insert HTML code and edit it in a WYSIWYG environment.
+A template can feature a wide variety of blocks and sections reflecting desired email design.
+By choosing a template and removing the unnecessary blocks beautiful emails can be created very efficiently.
+
+#### Modifying Legacy template
+The GrapesJs Builder looks at the template's config file content before a template is displayed for selection.
+As of Mautic 3.3 a new line has to be added, that defines which email designer is the template suitable for:
```

Comment:
```suggestion
Since [Mautic `3.3`][mautic-3.3] a new line has to be added which defines the compatible Builders:
```

Following the styleguide: https://docs.mautic.org/en/contributing#linking-to-release-versions - macro already exists